schedule/functional/extra_tests_gnome_sdk.yaml

```diff
@@ -0,0 +1,9 @@
+name:   extra_test_gnome_sdk
+description:    >
+    Maintainer: jrauch
+    Extra tests about software in sdk on gnome
+schedule:
+    -boot/boot_to_desktop
+    -console/prepare_test_data
```

Comment:
We should in the future schedule those "preparation" tests in the create_hdd_* scenarios, to not repeat them for each extra tests testsuite.

Reply:
That would make sense, but is outside the scope of this ticket